contentctl/actions/detection_testing/views/DetectionTestingViewCLI.py

```diff
@@ -51,6 +51,8 @@ def showStatus(self, interval: int = 1):
         while True:
             summary = self.getSummaryObject()
 
+            # TODO (cmcginley): there's a 1-off error here I think (we show one more than we
+            #   actually have during testing)
             total = len(
```

Comment:
This has been a longstanding issue that. I think, was initially added to avoid implementing lots of logic to avoid divide-by-zero errors.
Let's fix this once and for all as it's been an annoyance for a long time.

Reply:
https://github.com/splunk/contentctl/issues/338